### PR TITLE
TELCODOCS:600 - document new features for OSC release notes

### PR DESCRIPTION
**Pull request by MirzWeiss**
Version(s): Just 4.11

Issue:
[TELCODOCS-600](https://issues.redhat.com/browse/TELCODOCS-600)

[Link to docs preview](http://file.emea.redhat.com/miweiss/TELCODOCS-600/sandboxed_containers/sandboxed-containers-4.11-release-notes.html)

Acked by SMEs and QE.

### CODE REVIEW

**Comment by cpmeadors** [comment]
QE ACK

**Comment by tbuskey** [comment]
QE ACK AWS baremetal

File: sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc [hunk 1]
```diff
@@ -16,7 +16,24 @@ This product is fully supported and enabled by default as of {product-title} 4.1
...
+
+[id="osc-baremetal-fully-supported-on-aws"]
+=== {sandboxed-containers-first} availability on AWS bare metal
+
+Previously, {sandboxed-containers-first} availability on AWS bare metal was in Technology Preview. With this release, installing {sandboxed-containers-first} on AWS bare-metal clusters is fully supported.
+
+[id="osc-with-sno-topology"]
+=== Support for {sandboxed-containers-first} on {sno}
+
+{sandboxed-containers-first} now work on {sno} clusters when the {sandboxed-containers-operator} is installed by {rh-rhacm-first}.
```
Comment (jensfr) [inline sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc:36]: @MirzWeiss Actually the SNO cluster is installed by RHACM and then the operator is installed as usual.

Comment (MirzWeiss) [reply sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc:36]: @jensfr , @reihl had said the operator is also installed by RHACM... You're saying only the SNO cluster is installed by RHASCM, and then the operator is installed as usual by the user?

Comment (jensfr) [reply sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc:36]: @MirzWeiss @reihl I thought that's what Rei rold @abhbaner :-)  I never uses RHACM myself, when I tested sno I used a cluster that was created in a different way. I'm confused, but I'll trust Rei when he says that RHACM installs the operator

Comment (reihl) [inline sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc:36]: /approve

File: sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc [hunk 2]
```diff
@@ -16,7 +16,24 @@ This product is fully supported and enabled by default as of {product-title} 4.1
 [id="sandboxed-containers-1-3-new-features-and-enhancements"]
 == New features and enhancements
 
+[id="refined-kata-metrics"]
+=== Container ID in metrics list
 
+The `sandbox_id` with the ID of the relevant sandboxed container now appears in the metrics list in the *Metrics* page in the web console.
```
Comment (jldohmann) [inline sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc:22]:
```suggestion
The `sandbox_id` with the ID of the relevant sandboxed container now appears in the metrics list on the *Metrics* page in the web console.
```

**Review by jldohmann — COMMENTED** [review]
Small nit, otherwise LGTM!

**Review by jldohmann — APPROVED** [review]

**Review by littlejawa — APPROVED** [review]
lgtm

**Review by jensfr — APPROVED** [review]
lgtm

**Pull request closed without merging**